packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -122,14 +143,19 @@ function mockFetchWithDynamicResponse(dataForUrl: any) {
 }
 
 const MOCK_PRFERENCES = { state: { selectedAddress: 'foo' } };
-const PROVIDER = new HttpProvider(
+const GOERLI_PROVIDER = new HttpProvider(
   'https://goerli.infura.io/v3/341eacb578dd44a1a049cbc5f6fd4035',
 );
 const MAINNET_PROVIDER = new HttpProvider(
   'https://mainnet.infura.io/v3/341eacb578dd44a1a049cbc5f6fd4035',
 );
+const PALM_PROVIDER = new HttpProvider(
+  'https://palm-mainnet.infura.io/v3/3a961d6501e54add9a41aa53f15de99b',
```

Comment:
Curious why this was added - was this just to make the "MOCK_CUSTOM_NETWORK" more "custom" by setting it to a non-Ethereum chain?

Reply:
yeah exactly.